ipwb/replay.py

```diff
@@ -315,6 +315,9 @@ def queryTimeGate(urir):
     if adt is None:
         adt = ipwbUtils.getRFC1123OfNow()
 
+    if not ipwbUtils.isRFC1123Compliant(adt):
+        abort(400)
+
```

Comment:
No need to introduce `isRFC1123Compliant` function. We can wrap the following line in a `try` block to catch `ValueError` exception instead.

Reply:
See response to other comment.